src/main/java/com/blackduck/integration/create/apigen/generation/finder/ClassNameManager.java

```diff
@@ -5,7 +5,7 @@
  *
  * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
  */
-package com.synopsys.integration.create.apigen.generation.finder;
+package com.blackduck.integration.create.apigen.generation.finder;
```

Comment:
I would recommend changing the CORE_PACKAGE static in this file even if we don't do a generation right now.